File: morning_exercises/day7.py
def containsDuplicates(nums: list)->bool:
    seen = set()

    for num in nums:
        if num in seen: return True
        seen.add(num)
    
    return False

# print(containsDuplicates([1,2,3,4,1]))

def containsDuplicateCleanest(nums: list) -> bool:
    return len(nums) != len(set(nums))

print(containsDuplicateCleanest([1,2,3,4,1]))

File: morning_exercises/day7.cs
using System.Globalization;

public static class FindDuplicates
{
    public static bool containsDuplicates(List<int> nums)
    {
        var seen = new HashSet<int>();

        foreach (int num in nums)
        {
            if (seen.Contains(num)) return true;
            seen.Add(num);
        }
        return false;
    }
}